Assets/TabMasterScript.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.Collections;
```

Comment:
Rename to `TabManager`, `Master` is usually avoid nowadays due to it's connotation, also remove the word `Script` as having it as a `.cs` file implies that it is a script, to rename classes in Unity is a pain, I can't find my tried and true forum post for doing so but this one seems to be the same: http://www.cwgtech.com/renaming-c-classes-and-unity/